sources/world/world_countries_v7.hjson

```diff
@@ -44,16 +44,24 @@
       type: id
       name: iso2
       desc: Two letter abbreviation
+      regex: ^[A-Z]{2}$
+      alias: [
+        (geo\.){0,}country_iso_code$
```

Comment:
can we add a 2nd entry? (and add it to the `name` field

`(country|countries)`

This was raised here https://github.com/elastic/kibana/pull/99553.

It would also help to validate the functionality with multiple aliases. 

e.g. A field with very tight pattern-validation (e.g. `iso2`),  can be more "lenient" with field-aliases, since the we can explicitly validate the values.

It's something that needs to be handled on the Kibana side (still iterating).

Reply:
> can we add a 2nd entry? (and add it to the `name` field
> 
> `(country|countries)`
> 
> This was raised here [elastic/kibana#99553](https://github.com/elastic/kibana/pull/99553).
> 
> It would also help to validate the functionality with multiple aliases.
> 
> e.g. A field with very tight pattern-validation (e.g. `iso2`), can be more "lenient" with field-aliases, since the we can explicitly validate the values.
> 
> It's something that needs to be handled on the Kibana side (still iterating).

Discussed offline with Thomas and I will add the suggested aliases.

Reply:
> I added this test here to check the data. Is that what you mean?

:facepalm: yeah, that's exactly what I meant :sweat_smile: